### ISSUE TITLE
Using rkcommon with openvkl require to use LD_LIBRARY_PATH for loading

### ISSUE THREAD

**Issue by mwestphal** [openvkl/openvkl#17]
When loading openvkl with rkcommon and ospray, I have the following issue:

```
[openvkl] INITIALIZATION ERROR: could not open module lib openvkl_module_cpu_device: libopenvkl_module_cpu_device_4.so.1: cannot open shared object file: No such file or directory

[openvkl] WARNING: unrecognized device type 'cpu_8'.
[openvkl] INITIALIZATION ERROR: Could not find device of type: cpu_8.  Make sure you have the correct VKL libraries linked.
```

Setting LD_LIBRARY_PATH to libopenvkl_module_cpu_device_4.so location fixes the issue but it should not be needed.

**Comment by gregjohnson** [comment]
This should be fixed with the v1.3.2 release, made just now. Please let us know if you still have issues.

**Comment by mwestphal** [comment]
I'm using ospray superbuild, so this will take a while before I can actually test this. Thanks for the fix :)
